CougSat1-ADCS/src/drivers/DRV8837.h

```diff
@@ -16,22 +16,24 @@
 #ifndef DRV8837_H_
 #define DRV8837_H_
 
-#include <mbed.h>
+#include "mbed.h"
 #include "ADCSPins.h"
+#include "../tools/CISError.h"
```

Comment:
Have the relevant imports when needed. This header file does not use CISError at all so move it to an include in the source